test/data/dataNoTrimCRLF

```diff
@@ -1,3 +1,3 @@
-name,age
-joe,20
+name,age
+joe,20

```

Comment:
This file didn't actually have CRLF line endings. I assume that was accidental?

Reply:
hmm... This was weird. But will change to CRLF...